pkg/services/oracle/neofs/neofs.go

```diff
@@ -45,41 +47,49 @@ var (
 // URI scheme is "neofs:<Container-ID>/<Object-ID/<Command>/<Params>".
 // If Command is not provided, full object is requested.
 func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string) (io.ReadCloser, error) {
-	objectAddr, ps, err := parseNeoFSURL(u)
-	if err != nil {
-		return nil, err
-	}
-
-	c, err := client.New(client.PrmInit{})
+	c, err := GetSDKClient(ctx, addr)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create client: %w", err)
+		return clientCloseWrapper{c: c}, fmt.Errorf("failed to create client: %w", err)
 	}
+	return GetWithClient(ctx, c, priv, u, true)
+}
 
-	var (
-		res  = clientCloseWrapper{c: c}
-		prmd client.PrmDial
-	)
-	prmd.SetServerURI(addr)
-	prmd.SetContext(ctx)
-	err = c.Dial(prmd) //nolint:contextcheck // contextcheck: Function `Dial->Balance->SendUnary->Init->setNeoFSAPIServer` should pass the context parameter
+// GetWithClient returns a neofs object from the provided url using the provided client.
+// URI scheme is "neofs:<Container-ID>/<Object-ID/<Command>/<Params>".
+// If Command is not provided, full object is requested.
+func GetWithClient(ctx context.Context, c *client.Client, priv *keys.PrivateKey, u *url.URL, needClientCloser bool) (io.ReadCloser, error) {
+	objectAddr, ps, err := parseNeoFSURL(u)
 	if err != nil {
-		return res, err
+		return nil, err
 	}
+	s := user.NewAutoIDSignerRFC6979(priv.PrivateKey)
+	var result io.ReadCloser
 
-	var s = user.NewAutoIDSignerRFC6979(priv.PrivateKey)
 	switch {
```

Comment:
Useless blank line, you split switch-related variable from the switch itself. And unify two variables deinition, it's either `var` or `:=`.